Userland/Libraries/LibVideo/Containers/Matroska/Reader.h

```diff
@@ -25,6 +25,8 @@ class Reader {
     typedef Function<DecoderErrorOr<IterationDecision>(TrackEntry const&)> TrackEntryCallback;
 
     static DecoderErrorOr<Reader> from_file(StringView path);
+    static DecoderErrorOr<Reader> from_mapped_file(NonnullRefPtr<Core::MappedFile> const& mapped_file);
```

Comment:
same here, no const& for this, since we want to share ownership with the new Reader object. That means that the Reader function should do something like 

```
auto reader = TRY(from_data(mapped_file->bytes()));
reader.m_mapped_file = move(mapped_file);
return reader;
```
as well.